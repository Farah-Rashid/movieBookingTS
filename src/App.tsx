import React from 'react';
import HomePage from './components/HomePage';
import { Routes, Route } from "react-router-dom"
import BookingSeats from './components/BookingSeats';
function App() {
  return (
    <>
      <Routes>
        <Route path="/" element={<HomePage />} />
        <Route path="book/:id/:title" element={<BookingSeats />} />
      </Routes>
    </>
  );
}

export default App;
