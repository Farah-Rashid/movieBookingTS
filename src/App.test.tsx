import App from './App';
import { BrowserRouter } from 'react-router-dom'
import { render } from '@testing-library/react'
describe("testing movie cards", () => {
  test('should take a snapshot', () => {
    const { container } = render(<BrowserRouter><App /></BrowserRouter>)
    expect(container).toMatchSnapshot();
  })


})