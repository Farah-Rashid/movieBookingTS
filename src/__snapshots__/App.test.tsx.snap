// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing movie cards should take a snapshot 1`] = `
<div>
  <header
    class="sc-hKMtZM kUfBkT"
  >
    <h1>
      Book Tickets
    </h1>
    <form>
      <input
        placeholder="search"
        type="text"
        value=""
      />
    </form>
  </header>
  <article
    class="sc-jSMfEi gciWvL"
  >
    <h2>
      Sorry, there is no result for keyword you searched.
    </h2>
    <img
      alt="no img found"
      src="EmptyState.png"
    />
  </article>
</div>
`;
